src/Cake.Common/Tools/DotNetCore/Execute/DotNetCoreExecutor.cs

```diff
@@ -40,7 +40,6 @@ public void Execute(FilePath assemblyPath, string arguments, DotNetCoreSettings
             {
                 throw new ArgumentNullException("assemblyPath");
             }
-
```

Comment:
@patriksvensson shouldn't StyleCop be complaining here?  I thought a new line was required after every closing bracket?